sdk/core/azure-core/azure/core/credentials.py

```diff
@@ -28,3 +29,45 @@ def get_token(self, *scopes, **kwargs):
     from collections import namedtuple
 
     AccessToken = namedtuple("AccessToken", ["token", "expires_on"])
+
+__all__ = ["AzureKeyCredential"]
```

Comment:
What's this for? `AccessToken` is public.

Reply:
Then `AccessToken` should also be in `__all__`

Reply:
added AccessToken to the __all__ in credentials.py